docs/shared/guides/dynamic-mfe-angular.md

```diff
@@ -0,0 +1,167 @@
+
+This guide will walk through how the concept of “Build once, deploy everywhere” can be easily achieved in a Micro Frontend Architecture that uses Dynamic Module Federation.
+
+## Prerequisites
+
+This guide follows the previous Angular Micro Frontend guide. It will use the same workspace that was created in that guide.
+If you have not completed it, you can find that guide [here](/guides/setup-mfe-with-angular).
+
+## Aim
+
+The aim of this guide is two-fold. We want to be able to
+Transform an existing Static Module Federation setup to use Dynamic Federation.
+To generate a new Micro Frontend application that uses Dynamic Federation.
+
+## What we’ll build
+
+To achieve the aims, we will do the following:
+Change the Dashboard application to use Dynamic Federation
+Generate a new Employee Dashboard application that will use Dynamic Federation
+It should use the existing Login application
+It should use a new Todo application
+
+## Converting the Dashboard application
+
+The Dashboard application is a Host application that loads the Remote applications at runtime based on their deployed locations when the application was built.
+
+We want to change this so that the Dashboard application can make a network request at runtime to find out the deployed locations of the Remote applications.
+
+There are 3 steps involved with this:
+Make a network request to fetch the locations of the Remote applications _(the Remote Definitions)_.
+Set the Remote Definitions so that webpack is aware of how to find the Remote applications when requested.
+Change how we load the Remote applications so that webpack can fetch any required files correctly.
+
+### Fetch and Set the Remote Definitions
+
+Perhaps one of the easiest methods of fetching the Remote Definitions at runtime is to store them in a JSON file that can be present in each environment. The Host application then only has to make a GET request to the JSON file.
+
+We’ll start by creating this file. Add a `mfe.manifest.json` file to the `src/assets/` folder in our Dashboard application with the following content:
+
+```json
+{
+  "login": "http://localhost:4201"
+}
+```
+
+Next, open `main.ts` under the `src/`folder and replace it with the following:
+
+```ts
+import { setRemoteDefinitions } from '@nrwl/angular/mfe';
+
+fetch('/assets/mfe.manifest.json')
+  .then((res) => res.json())
+  .then((definitions) => setRemoteDefinitions(definitions))
+  .then(() => import('./bootstrap').catch((err) => console.error(err)));
+```
+
+You’ll notice that we:
+Fetch the JSON file
+Call `setRemoteDefinitions` with the contents of the JSON file
+This tells webpack where each of our Remote applications has been deployed to!
+
+### Change how Remotes are loaded
+
+At the moment, webpack is statically building our application, telling it at build time where our Remotes are. That is because they are specified in the `mfe.config.js` file.
```

Comment:
```suggestion
At the moment, Webpack is statically building our application, telling it at build time where our Remotes are. That is because they are specified in the `mfe.config.js` file.
```